server/chat-plugins/mafia.ts

```diff
@@ -151,12 +152,26 @@ function writeFile(path: string, data: AnyObject) {
 MafiaData = readFile(DATA_FILE) || {alignments: {}, roles: {}, themes: {}, IDEAs: {}, terms: {}, aliases: {}};
 if (!MafiaData.alignments.town) {
 	MafiaData.alignments.town = {
-		name: 'town', plural: 'town', memo: [`This alignment is required for the script to function properly.`],
+		name: 'Town',
+		plural: 'Town',
+		memo: [`This alignment is required for the script to function properly.`],
 	};
 }
 if (!MafiaData.alignments.solo) {
 	MafiaData.alignments.solo = {
-		name: 'solo', plural: 'solo', memo: [`This alignment is required for the script to function properly.`],
+		name: 'Solo',
+		plural: 'Solo',
+		memo: [`This alignment is required for the script to function properly.`],
+	};
+}
+if (!MafiaData.themes.nominations) {
+	MafiaData.themes.nominations = {
+		name: `Nominations`,
+		desc: `Every odd night, the Mafia select three players. The next day, you can only vote for one of these three players.`,
+		7: `Mafia Goon, Mafia Goon, Villager, Villager, Villager, Villager, Villager`,
+		11: `Mafia Goon, Mafia Goon, Mafia Goon, Villager, Villager, Villager, Villager, Villager, Villager, Villager, Villager`,
+		15: `Mafia Goon, Mafia Goon, Mafia Goon, Mafia Goon, Villager, Villager, Villager, Villager, Villager, Villager, Villager, Villager, Villager, Villager, Villager`,
+		19: `Mafia Goon, Mafia Goon, Mafia Goon, Mafia Goon, Mafia Goon, Villager, Villager, Villager, Villager, Villager, Villager, Villager, Villager, Villager, Villager, Villager, Villager, Villager, Villager`,
 	};
 }
```

Comment:
Why does nommy need to be hardcoded here, isn't theme data usually just located in the appropriate JSON file?